### ISSUE TITLE
Turn permissions article into templated data like grades

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#2496]
We have a couple articles that could have more detail or may need adjustment after the next release.

https://www.thecrag.com/article/IndexPermissions

https://www.thecrag.com/article/SensitiveCrags

Ideally I would like to have the IndexPermissions article be given a matrix of types of editing tasks and we can format it as a table showing what exactly what each roles can do what, and also how they change when a crag is locked #2495  or sensitive #2476.

**Comment by scd** [comment]
I have added stuff to the sensitive crags article. I don't think there is any point in templatising this one just yet. I would like to see some view levels in action first. I have also thrown this template into the translations db. Every time I touch a template I plan to translate it.

Index permissions needs a total rewrite based on the new business rules in:

https://docs.google.com/document/d/1NGNvNOZwjQQeX2hO9-RcrVGid-KxHA6nUViRuWkKx3E/edit#

Don't want to do that until we have agreed the business rules and I have made some implementation changes. This may not make the next release.
